unlock-app/src/components/interface/pagination/PageNumbers.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { PageGroup, PageNumber, PageNumberActive } from './Pagination'
+
+const PageNumbers = ({ numberOfPages, currentPage, goToPage }) => {
+  const pageNumber = number => {
+    return number === currentPage ? (
+      <PageNumberActive onClick={() => goToPage(number)}>
+        {number}
+      </PageNumberActive>
+    ) : (
+      <PageNumber onClick={() => goToPage(number)}>{number}</PageNumber>
+    )
+  }
+
+  // If number of pages less than 10, show all 10 page numbers
+  if (numberOfPages <= 10) {
+    return (
+      <PageGroup>
+        {Array(numberOfPages)
+          .fill()
+          .map((_, pn) => {
+            return pageNumber(pn + 1)
+          })}
+      </PageGroup>
+    )
+  }
+
+  // If number of pages greater than 10,  show first five and last five (1 2 3 4 5 .... 26 27 28 29 30)
+  return (
+    <PageGroup>
+      {//show first five page numbers
+        Array(5)
```

Comment:
Same here, please use a constant so it is really easy to change to another number should we need to!